### ISSUE TITLE
Optimize cookie configuration

### ISSUE THREAD

**Issue by aschempp** [contao/contao-manager#81]
We're using cookies for the JWT and XSRF token. The configuration should be optimized.

https://github.com/contao/contao-manager/blob/develop/api/Security/JwtManager.php#L101-L103
https://github.com/contao/contao-manager/blob/develop/api/Security/JwtManager.php#L135-L145

@contao/developers any hints on what would be best? Limit to current domain, to the Phar file only, etc?

**Comment by Toflar** [comment]
I think I'd use this:

```php
$expire = new \DateTime();
$expire->add(new \DateInterval('PT2H')); // 2 hours

new Cookie(
    self::COOKIE_XSRF,
    $token,
    $expire,
    '/',
    $request->getHttpHost(),
    $request->isSecure(),
    false,
    false,
    Cookie:: SAMESITE_STRICT // Most important change
)
```

**Comment by aschempp** [comment]
Implemented in 59f83b13c4d8da8a0683dcebdc99eafe728db5b5
